tcf_website/templates/reviews/review_form_content.html

```diff
@@ -40,7 +40,7 @@ <h2 class="h4 review-form-header form-group col-12 my-4">
     Tell us more about the course and instructor</h2>
   <div class="form-group col mb-3">
     <textarea class="form-control" id="reviewtext" name="text" rows="5"
-              placeholder="(Aim for 3 sentences or more!)" minlength="100" maxlength="5000">{{ form.text.value }}</textarea>
+              placeholder="(Aim for 300 words or more! Some suggestions on what to write about are information about the professor's teaching style and ability to convey information, workload, assignment types and/or anything you think is important to know when taking the class. Include what you liked about the course and what could be improved on. Well-written reviews help other students get a good understanding of classes they're considering taking!)" minlength="100" maxlength="5000">{{ form.text.value }}</textarea>

```

Comment:
Actually, did we ever have a discussion on whether we want to push for "300 words"? That seems like a short paragraph and might deter people from starting a review.

Reply:
I don't think so, we should discuss it tomorrow. The reason I included it in the placeholder is because of the progress bar.

Reply:
Whoops, looks like we forgot to discuss this last week - I'll add it to the agenda for this week, but in the future don't forget to put items on the agenda!